roles/rsyslog/tasks/main.yml

```diff
@@ -78,8 +78,10 @@
 
         - name: Archive the contents of {{ __rsyslog_config_dir }} to
             the backup dir
+          vars:
+            __now: "{{ lookup('pipe', 'date +%s') }}"
           command: >
-            tar -cvzPf "{{ __rsyslog_backup_dir }}/backup.tgz"
+            tar -cvzPf "{{ __rsyslog_backup_dir }}/backup{{ __now }}.tgz"
```

Comment:
lookup will return the timestamp on the controller, not on the managed host where the backup will be created.  Is there an ansible fact which contains the current time on the managed host?

Reply:
I guess you can't use `ansible_date_time.iso8601_basic_short` here because the method below relies on `__now` being in seconds?  here are the time fields in ansible_date_time`:
```
    "epoch": "1641649779",
    "hour": "08",
    "iso8601": "2022-01-08T13:49:39Z",
    "iso8601_basic": "20220108T084939145434",
    "iso8601_basic_short": "20220108T084939",
    "iso8601_micro": "2022-01-08T13:49:39.145434Z",
```
I think you could use `iso8601_basic_short` here - it gives the same second resolution as `date +%s`, and I think it sorts/collates the same way